whiteBoarding/fizzbuzz/fizzbuzz.md

```diff
@@ -0,0 +1,5 @@
+`1st) return fizzbuzz. 2nd) return fizzbuzz but in lieu of fizz,buzz,fizzbuzz return key values from this provided object => let sounds = {
```

Comment:
in the other files the instructions for the exercises are in a file called `instructions.md`.  Also these files have an outline of what needs to be done and a title and links to attempts.  Structure within files is often there for a reason.